lib/internal/process/execution.js

```diff
@@ -61,7 +60,7 @@ function getEvalModuleUrl() {
  */
 function evalModuleEntryPoint(source, print) {
   if (print) {
-    throw new ERR_EVAL_ESM_CANNOT_PRINT();
+    source = `console.log((${source}))`;
```

Comment:
```suggestion
    source = `console.log(await (async(() => {${source}})()))`;
```
will wrapping source with an async iife make [multiple expressions](https://github.com/nodejs/node/pull/52105#issuecomment-2000451468) and missing [await](https://github.com/nodejs/node/pull/52105#issuecomment-2000426507) work?

Reply:
It would force the user to write `return`, which would be weird (let’s not forget that using `return` at the top-level of a module is would throw a parse error)